### PR TITLE
Add exit attached container with port mapping test

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

**Comment by hmahmood** [comment]
lgtm

**Comment by anchal-agrawal** [comment]
LGTM

File: tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot [hunk 1]
```diff
@@ -54,4 +54,58 @@ Create container without specifying host port
...
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm -f $(docker ${params} ps -aq)
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  mkfifo /tmp/fifo1
+    ${result}=  Start Process  docker ${params} run -i --name ctr1 -p 1900:9999 -p 2200:2222 busybox /bin/top < /tmp/fifo1  shell=True  alias=sh1
+    Sleep  5
+    ${rc}  ${output}=  Run And Return Rc And Output  echo q > /tmp/fifo1
+    ${result}=  Wait for process  sh1
+    Log  ${result.stdout}
+    Log  ${result.stderr}
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot]: we need some sort of verification here. at least RC check if nothing else, but preferably something else :)

File: tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot [hunk 2]
```diff
@@ -54,4 +54,58 @@ Create container without specifying host port
...
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ctr3
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+    # PowerOff VM out-of-band
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc vm.power -off ${vch-name}/"ctr3*"
+    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc vm.power -off "ctr3*"
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Be Equal As Integers  ${rc}  0
```
Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot]: This procedure is now done in enough places, I would like to see this as a keyword in Util.robot. Can you move that over please?

Comment (andrewtchin) [inline tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot]: fixed

File: tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot (hunk 1 above)
Comment (andrewtchin) [inline tests/test-cases/Group1-Docker-Commands/1-25-Docker-Port-Map.robot]: fixed

**Pull request merged**